src/modules/feature-modules/assessment/pages/innovation/assessment/assessment-edit.component.ts

```diff
@@ -199,4 +199,9 @@ export class InnovationAssessmentEditComponent extends CoreComponent implements
 
   }
 
+  getFormChanges(form: any): void {
```

Comment:
2 notes:
First, the name of this method shoould be something like "on...". Ex: "onFormValuesChange" or "onFormChange". This helps when reading the code, that this method is called by an "event".

Second: the use of "any" should be used with caution. In this case, why do you need that variable?